cli/stream_command.go

```diff
@@ -672,16 +672,15 @@ func (c *streamCmd) viewAction(_ *fisk.ParseContext) error {
 		}
 	}()
 
-	shouldTerminate := false
-
 	for {
 		msg, last, err := pgr.NextMsg(ctx)
 		if err != nil {
 			if !last {
 				return err
 			}
 			// later we know we reached final last after showing the final message
-			shouldTerminate = true
+			log.Println("Reached apparent end of data")
+			return nil
```

Comment:
This is how it was before but that’s actually a bug - note the last message is never rendered we bail before rendering in cases where pending ==. 0